assets/scss/components/citadel/cards/_cards.scss

```diff
@@ -49,7 +49,7 @@
 .card-image {
     @include lazy-loaded-img;
     border: 0;
-    width: auto;
+    width: 100%;
```

Comment:
more explicit CSS sizing is helpful for lazysizes - this prevents page flickering.